a09.evaluator/Program.cs

```diff
@@ -1,21 +1,37 @@
-﻿namespace Eval;
-
-class Program {
-   static void Main (string[] args) {
-      var eval = new Evaluator ();
-      for (; ; ) {
-         Console.Write ("> ");
-         string text = Console.ReadLine () ?? "";
-         if (text == "exit") break;
-         try {
-            double result = eval.Evaluate (text);
-            Console.ForegroundColor = ConsoleColor.Green;
-            Console.WriteLine (result);
-         } catch (Exception e) {
-            Console.ForegroundColor = ConsoleColor.Yellow;
-            Console.WriteLine (e.Message);
+﻿// ---------------------------------------------------------------------------------------
+// Training ~ A training program for new interns at Metamation, Batch - July 2023
+// Copyright (c) Metamation India.                                              
+// ---------------------------------------------------------------------------------------
+// Program.cs
+// Implement the ParseDouble function. Here’s an example function that does the simpler
+// job of parsing an integer from forms like 12, -123, +1234. Use this as a way of getting started.
+// This assignment is very important, please do try to create a solution.
+// ---------------------------------------------------------------------------------------
+using static System.Console;
+namespace Eval {
+   #region Program --------------------------------------------------------------------------------
+   /// <summary>Expression Evaluator</summary>
+   class Program {
+      #region Method ------------------------------------------------
+      /// <summary>Evaluate the expressions</summary>
+      static void Main () {
+         var eval = new Evaluator ();
+         for (; ; ) {
+            Write ("> ");
+            string input = ReadLine () ?? "".Trim ().ToLower ();

```

Comment:
If the input is 'Exit', then the program should be exited irrespective of upper/lower case.